app/feed/tests/test_feed_routes.py

```diff
@@ -0,0 +1,87 @@
+from flask import url_for
+import pytest
+
+from app.common.tests.utils import is_none_or_type
+from app.factories import ScoresFactory
+
+
+@pytest.mark.integration
+def test_feed(client_with_user_and_header):
+    client, _, session_header, _ = client_with_user_and_header
+    score = ScoresFactory()
+    response = client.get(
+        url_for("scoring.get_feed"),
```

Comment:
I would have expected this to be "feed.get_feed", but `feed/routes.py` does actually use the Blueprint from `app.scoring` https://github.com/ClimateMind/climatemind-backend/blob/develop/app/feed/routes.py#L4

Reply:
@brianpeiris I'll create a separate issue